### PR TITLE
[IMP] Quality: Pass - Fail check Shop Floor

### PR DESCRIPTION
**Pull request by hojo-odoo**
This PR replaces the tablet view workflow in the Pass - Fail check doc with a new section detailing the workflow for the Shop Floor module.

### CODE REVIEW

File: content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst
Comment (Felicious) [inline content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst:114]: If you happen to review this PR before seeing my comments in the [instructions quality check PR](https://github.com/odoo/documentation/pull/6779#discussion_r1436558331), here's a recap:

- we use italics when defining concepts or Odoo terms for the very first time. 
- However, the times after, we probably don't need to capitalize pass-fail anymore. However, each app and term is different, so if you strongly disagree with that, feel free to let me know :smile:

File: content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst
Comment (Felicious) [inline content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst:5]: Also, I'll just make a comment here since I feel like I'm not really trying on my reviews if I leave too few comments :sweat_smile: 

This comment is an extreme nitpick, so feel free to ignore! While skimming your doc, I noticed that you didn't use any admonition blocks. I'm a sucker for colors so I wanted to suggest an opportunity to use one on lines 102 - 104, where it says

> For a complete guide on how to fill out quality
alert forms, view the documentation on :ref:`quality alerts
<quality/quality_management/quality-alerts>`.

What do you think about using a seealso admonition block instead (also using the doc tag with a relative tag to link to internal documentation):

```
.. seealso::
   :doc:`Complete guide on quality alerts <../quality_management/quality_alerts>`
```

**Review by Felicious — APPROVED** [review]
Great job with this Shop Floor update, @hojo-odoo!! I just had minor comments on formatting. Everything looked good when I tested the flow!

File: content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst:117]:
```suggestion
*Pass - Fail* quality checks configured for work orders **must** be completed from the *Shop Floor*
module. To do so, begin by navigating to :menuselection:`Manufacturing --> Operations -->
Manufacturing Orders`. Select an |MO| that includes a work order for which a *Pass - Fail* quality
check is required.
```

File: content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst:126]:
```suggestion
center where the order is configured to be processed, and isolates the work order's card, so no
other cards are shown.
```

File: content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst:135]:
```suggestion
If the :guilabel:`Pass` button is clicked, the pop-up window moves to the next step for the work
order. If the :guilabel:`Fail` button is clicked, a :guilabel:`Quality Check Failed` pop-up window
appears, detailing what should be done next.
```

**Review by ksc-odoo — APPROVED** [review]
Alrighty @hojo-odoo Just finished my Peer Review of this doc, and apart from a couple SUPER-MINOR adjustments, I think this one is ready to move on to the next stage. Once those little changes are addressed, it should be good-to-go 👍

**Comment by hojo-odoo** [comment]
@samueljlieber This is ready for technical review.

**Review by samueljlieber — APPROVED** [review]
Looks good to me @hojo-odoo! Nice job on the improvements in this PR. 

I have only one correction, and it is to the commit tag & PR title, please update it to follow the [naming convention](https://github.com/odoo/documentation/blob/saas-16.4/commit_template.txt) as well as make it a bit shorter/concise, something like:
```
[IMP] quality: pass-fail check shop floor
```

I am approving now, but please update the commit tag & PR title before tagging for merge 🙂

File: content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst
Comment (StraubCreative) [reply content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst:5]: I second this— the blocks are useful for readers in a lot of ways and colors are nice!

File: content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst:120]: [reference](https://fontawesome.com/v5/icons/external-link?f=classic&s=solid)
```suggestion
(external link icon)` button on the line of the work order to be processed. On the
```

File: content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst
Comment (StraubCreative) [inline content/applications/inventory_and_mrp/quality/quality_check_types/pass_fail_check.rst:141]: would surround this in a **tip** since it's supplemental content

**Review by StraubCreative — APPROVED** [review]
hi @hojo-odoo 
approving with a couple comments below
please address the icon name and consider using the admonition block at the end

@robodoo delegate=hojo-odoo

**Comment by hojo-odoo** [comment]
@robodoo r+

**Pull request closed without merging**
